geany.desktop.in

```diff
@@ -10,3 +10,4 @@ Terminal=false
 Categories=GTK;Development;IDE;
 MimeType=text/plain;text/x-chdr;text/x-csrc;text/x-c++hdr;text/x-c++src;text/x-java;text/x-dsrc;text/x-pascal;text/x-perl;text/x-python;application/x-php;application/x-httpd-php3;application/x-httpd-php4;application/x-httpd-php5;application/xml;text/html;text/css;text/x-sql;text/x-diff;
 StartupNotify=true
+Keywords=Text;Editor;
```

Comment:
Should be translatable (prefixed with a `_`)

https://specifications.freedesktop.org/desktop-entry-spec/latest/ar01s05.html
https://git.gnome.org/browse/gedit/tree/data/org.gnome.gedit.desktop.in.in#n19